db/blob/blob_file_builder.h

```diff
@@ -70,7 +72,9 @@ class BlobFileBuilder {
   uint64_t blob_file_size_;
   CompressionType blob_compression_type_;
   const FileOptions* file_options_;
+  int job_id_;
   uint32_t column_family_id_;
+  std::string column_family_name_;
```

Comment:
Nit: we probably can use `const std::string&` here because the column family should outlive this builder.